.travis.yml

```diff
@@ -28,7 +27,7 @@ jdk:
 sudo: false
 
 install:
-    - "pip install tox-travis"
+    - "pip install tox-travis tox==3.10.0"
```

Comment:
Why did you need to pin the version of tox?  This might be the issue.

The py27 travis build is complaining that it cannot find `importlib_metadata`:
```
ImportError: No module named importlib_metadata
```

`tox==3.10.0` does not define `importlib-metadata` as a dependency, but later versions of tox, such as `3.13.2`, do.  Try pinning this version, if pinning is needed at all.

Reply:
Pinning tox (along with pluggy) was an experiment to see if that would resolve my problem. [This build of travis](https://travis-ci.com/github/NCI-GDC/indexd/jobs/488673501) shows the same error before pinning tox. importlib_metadata is the next, in what I assume will be, a series of missing imports.

The reason I chose this version of tox was because it's what's installed on my pc and it works on my machine. I wanted to try to replicate as much as I could but it does not seem to be the answer.

I can try the version of tox you suggested.